components/prism-cmake.js

```diff
@@ -1,8 +1,18 @@
-Prism.languages.cmake = Prism.languages.extend('python', {
+Prism.languages.cmake = {
+    'comment': {
+		pattern: /(^|[^\\])#.*/,
+		lookbehind: true
+    },
+    'string': {
+		pattern: /(?:[rub]|rb|br)?("|')(?:\\.|(?!\1)[^\\\r\n])*\1/i,
+		greedy: true
+    },
+    'punctuation': /[()]/,
+    'number': /(?:\b(?=\d)|\B(?=\.))(?:0[bo])?(?:(?:\d|0x[\da-f])[\da-f]*\.?\d*|\.\d+)(?:e[+-]?\d+)?j?\b/i,
```

Comment:
Just a little question: Why apart from `cmake_minimum_required (VERSION 2.8.11)` are numbers used? Aren't [math expressions done via strings](https://cmake.org/cmake/help/v3.13/command/math.html)?

Reply:
In CMake everything is a string. The only numbers look like the one you pasted. Numbers are used for versioning mainly and are separated with a dot.

Reply:
Then should the more simple pattern: `\d+(?:\.\d+)*` also work?
The Python number pattern also handles binary, hexadecimal, and complex numbers and scientific notation. We don't need that, do we?

Reply:
Yes, we might use that. However, we need a bit more sophisticated pattern. It matches the 4 from e.g. CTEST_P4_CLIENT and somehow \r\n.
I will revisit this issue after examining other comments.

Reply:
My bad, I forgot the boundaries. The following should work.

```js
/\b\d+(?:\.\d+)*\b/
```